webprotege-gwt-ui-client/src/main/java/edu/stanford/bmir/protege/web/client/form/EntityFormStackPresenter.java

```diff
@@ -268,12 +272,16 @@ private void setMode(@Nonnull FormMode mode) {
     }
 
     private void commitEdits(@Nonnull OWLEntity entity) {
-        FormDataByFormId editedFormData = formStackPresenter.getFormData();
-        Collection<FormId> editedFormIds = editedFormData.getFormIds();
-        ImmutableMap<FormId, FormData> pristineFormData = pristineDataManager.getPristineFormData(editedFormIds);
-        dispatch.execute(new SetEntityFormsDataAction(projectId, entity, pristineFormData, editedFormData),
-                         // Refresh the pristine data to what was committed
-                         result -> updateFormsForCurrentEntity(ImmutableList.copyOf(editedFormIds)));
+        try {
+            FormDataByFormId editedFormData = formStackPresenter.getFormData();
+            Collection<FormId> editedFormIds = editedFormData.getFormIds();
+            ImmutableMap<FormId, FormData> pristineFormData = pristineDataManager.getPristineFormData(editedFormIds);
+            dispatch.execute(new SetEntityFormsDataAction(projectId, entity, pristineFormData, editedFormData),
+                    // Refresh the pristine data to what was committed
+                    result -> updateFormsForCurrentEntity(ImmutableList.copyOf(editedFormIds)));
+        }catch (Exception e) {
+            logger.info("An exception occured " + e);
```

Comment:
when "pristineDataManager.getPristineFormData(editedFormIds)" it can throw an exception that is not logged so it fails silently